app/helpers/file_manager_helper.rb

```diff
@@ -1,6 +1,6 @@
 module FileManagerHelper
-  def ocr_check(id)
-    return '✅' if FileSet.find(id).extracted_text
+  def ocr_check(presenter)
+    return '✅' if presenter.extracted_text
```

Comment:
Seems like we probably want to use the `extracted_text?` method here, or drop the method entirely